plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -360,9 +360,10 @@ public void updateTableStatistics(String databaseName, String tableName, AcidTra
             final Map<String, String> statisticsParameters = updateStatisticsParameters(table.getParameters(), updatedStatistics.getBasicStatistics());
             tableInput.setParameters(statisticsParameters);
             table = Table.builder(table).setParameters(statisticsParameters).build();
-            stats.getUpdateTable().call(() -> glueClient.updateTable(new UpdateTableRequest()
-                    .withDatabaseName(databaseName)
-                    .withTableInput(tableInput)));
+            Failsafe.with(CONCURRENT_MODIFICATION_EXCEPTION_RETRY_POLICY)
+                    .run(() -> stats.getUpdateTable().call(() -> glueClient.updateTable(new UpdateTableRequest()
```

Comment:
The retry should encompass `getExistingTable + GlueInputConverter.convertTable + updateStatisticsParameters`